### ISSUE TITLE
Optional id route parameter is lost when model binding for UmbracoApiController

### ISSUE THREAD

**Issue by p-m-j** [umbraco/Umbraco-CMS#11554]
### Which *exact* Umbraco version are you using? For example: 8.13.1 - don't just write v8

9.0.1

### Bug summary

Optional id route parameter is lost when model binding for UmbracoApiController

### Steps to reproduce

```csharp
using Microsoft.AspNetCore.Mvc;
using Umbraco.Cms.Web.Common.Controllers;

namespace Umbraco.Cms.Web.UI
{
    public class MyApiController : UmbracoApiController
    {
        public IActionResult Foo(int id) => Ok(id);

        public IActionResult Bar([FromRoute] int id) => Ok(id);
    }
}
```
+ Create a subclass of UmbracoApiController matching sample above
+ Navigate to /umbraco/api/MyApi/Foo/123
    + note 0 is rendered for response body
+ Navigate to /umbraco/api/MyApi/Bar/123
   + Note id param is rendered for response body

### Expected result / actual result

[FromRoute] should not be required but is, see https://docs.microsoft.com/en-us/aspnet/core/mvc/models/model-binding?view=aspnetcore-5.0

**Comment by p-m-j** [comment]
see umbraco\v9\src\Umbraco.Web.Common\Extensions\EndpointRouteBuilderExtensions.cs

**Comment by bjarnef** [comment]
Seems to be a duplicate of https://github.com/umbraco/Umbraco-CMS/issues/11523 🙂

**Comment by elit0451** [comment]
Fixed in #11557
